spyder/widgets/ipythonconsole/shell.py

```diff
@@ -70,6 +71,9 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         # To save kernel replies in silent execution
         self._kernel_reply = None
 
+        # To set the color of the matched parentheses
+        self.set_bracket_matcher_color_scheme(self.syntax_style)
```

Comment:
`set_color_scheme` does not seem to be called anywhere on startup. The initial settings are actually passed to the `JupytherWidget` as argument in the `super` call and the settings are applied directly within the `JupytherWidget`. However, these settings do not include the _matched parentheses color_, since the default hard-coded value is used. That is why we have to set it separately here in the `__init__` on our side.

So if we do not add this there, the setting to the  _matched parentheses color_ is applied only when there is actually a change to the color scheme after the Spyder is started and a call is made to `set_bracket_matcher_color_scheme`.